en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -170,9 +170,13 @@ This reference is defined by the [EKF2_HGT_REF](../advanced_config/parameter_ref
 
 ### Barometer
 
-Enable/disable using [EKF2_BARO_CTRL](../advanced_config/parameter_reference.md#EKF2_BARO_CTRL).
+Enable/disable using [EKF2_BARO_CTRL](../advanced_config/parameter_reference.md#EKF2_BARO_CTRL) as a source for [Height](#height) data.
 
-For more details about the configuration of height sources, [click here](#height).
+Note that data from only one barometer is fused, even if multiple barometers are available.
+The barometer with the highest [CAL_BAROx_PRIO](../advanced_config/parameter_reference.md#CAL_BARO0_PRIO) priority value is selected first, falling back to the next highest priority barometer if a sensor fault is detected.
```

Comment:
@bresch in discord it said first instance is detected, but given we have priorities I assume that we use them in the selection? If we do, how do we select between equal priorities?

Reply:
> given we have priorities I assume that we use them in the selection?

yes

> If we do, how do we select between equal priorities?

Then the first detected instance is used. This is kinda random so it's better to set the priorities properly.

Reply:
Thanks! I'll add  a note.